src/libraries/System.Data.Odbc/ref/System.Data.Odbc.cs

```diff
@@ -44,7 +45,7 @@ public sealed partial class OdbcCommand : System.Data.Common.DbCommand, System.I
         public override object? ExecuteScalar() { throw null; }
         public override void Prepare() { }
         public void ResetCommandTimeout() { }
-        object System.ICloneable.Clone() { throw null; }
+        object? System.ICloneable.Clone() { throw null; }
```

Comment:
These nullable changes are unrelated, are you OK committing in the same PR?

Reply:
Those came in from running /t:GenerateReferenceSource... validated the implementation and the implementation has those nullable annotations so I thought it was fine to include them to not forget all the annotation sthat needed to be exposed, but whichever you prefer I'm fine with.

Reply:
I'm OK with it.  FYI @roji

Reply:
Thanks for this @safern, I should have done this as part of the original annotation (but was unaware how to).

Should we have a build step to fail the CI for these kind of nullability discrepancies between ref and src?

Reply:
We do want to do that however our tooling to detect that (APICompat) is built on top of CCI so detecting this discrepancies is very hard because the nullable annotations can be expressed differently because they are stored in metadata so we can't do a token based comparison. We think it would be a better investment moving the tooling to use Roslyn and take advantage of the language APIs for this instead of having to implement ourselves every new language feature.

I have a prototype of this but haven't had funded time to finish that.

Reply:
Thanks for the update @safern, makes sense... FWIW I'm also suffering from the lack of APIs to expose NRT information in reflection (https://github.com/dotnet/runtime/issues/29723).